decisions/2025-06-17-RFC-target-data-metadata/target-data-schema.json

```
@@ -4,78 +4,113 @@
     "title": "Schema for Modeling Hub target data definitions",
     "description": "This is the schema of the target-data.json configuration file that defines metadata about target data used to visualise and evaluate modeling hub model outputs.",
     "type": "object",
-    "description": "Target data metadata.",
     "properties": {
-        "target_data_metadata": {
+        "schema_version": {
+            "description": "URL to a version of the Modeling Hub schema target-data-schema.json file (see https://github.com/hubverse-org/schemas). Used to declare the schema version a 'target-data.json' file is written for and for config file validation. The URL provided should be the URL to the raw content of the schema file on GitHub.",
+            "examples": [
+                "https://raw.githubusercontent.com/hubverse-org/schemas/main/v6.0.0/target-data-schema.json"
+            ],
+            "type": "string",
+            "format": "uri"
+        },
+        "observable_unit": {
+            "description": "Names of columns whose unique value combinations define the minimum observable unit across all target type data. Each combination of values must be unique (and in time-series data also unique across `as_of` data versions if applicable). The majority are expected to correspond to task ID names but may include other columns as well (e.g., the `date_col` column).",
+            "type": "array",
+            "uniqueItems": true,
+            "items": {
+                "type": "string"
+            }
+        },
+        "date_col": {
+            "description": "Name of the date column across hub data (time-series, oracle-output and ideally model-output). This is the column that stores the date on which observed data actually occurred.",
```

```suggestion
            "description": "Name of the date column across hub data (time-series, oracle-output and model-output (if present)). This is the column that stores the date on which observed data actually occurred.",
```
As not all hubs have a date column in the `model-output` and we don't want to enforce it, I think we should change "ideally" to something saying that it should be matching to the data column if there is any in model-output